routes/index.js

```diff
@@ -7,4 +10,21 @@ router.get('/', (req, res) => {
   res.render('index', {});
 });
 
+// log in with local strategy
+router.post('/login', passport.authenticate('local'),
```

Comment:
I get the following when trying to post to this route:
    Error: Unknown authentication strategy "local"
See comment in the app.js file. You never told passport to use LocalStrategy so 'local' is undefined